docs/dev/adding_instruments.rst

```diff
@@ -590,10 +677,26 @@ The code below shows how this can be accomplished with dynamic properties.
 
 
 Now our voltage property has a dynamic validity range, either [-1, 1] or [0, 1].
+A side effect of this is that the property's docstring should be less specific, to avoid it containing dynamically changed information (like the admissible value range).
```

Comment:
```suggestion
A side effect of this is, that the property's docstring should be less specific, to avoid it containing dynamically changed information (like the admissible value range).
```

Reply:
I don't agree with all comma suggestions, this one for example I'm pretty sure is non appropriate.